### DISCUSSION TITLE
adding kidney plasma clearance

### DISCUSSION THREAD

**Discussion by amaghrabi** [Open-Systems-Pharmacology/Forum#1116] [Questions & Problems]
![image](https://user-images.githubusercontent.com/80226127/152644115-0318996e-74ce-4914-9540-d91c3eb5f8ab.png)
hello everyone!! 
winch value i have to enter 
total plasma clearance OR  renal clearance 

thanks in advance

**Comment by Slow-Walker66** [comment]
I have the same problem.
Have you solved this?
Thanks

**Comment by StephanSchaller** [comment]
@amaghrabi , not sure what you are referring to in the screenshot (i.e. I don't see the label "total plasma clearance".

But to explain: the entered (and likely measured) "Plasma clearance" is recalculated to a "Specific clearance" parameter  which is the used in the model.

**Reply by Slow-Walker66** [reply]
Thanks for your reply.
I was confused by the yellow background chart. the "plasma clearance" is referring to the total plasma clearance of the drug, or only the kidney plasma clearance of the drug?
it's means when I want to input the measured data, I should input the “total plasma clearance” into the yellow chart or input the kidney plasma clearance one?
